addons/modules/stringtable.xml

```diff
@@ -218,6 +239,21 @@
             <Polish>Naprawianie pojazdu (Na leżąco)</Polish>
             <Japanese>車両の修理 (寝転がり)</Japanese>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleAmbientAnim_Dead">
+            <English>Dead</English>
+        </Key>
+        <Key ID="STR_ZEN_Modules_ModuleAmbientAnim_DeadLeaned">
+            <English>Dead (Leaned)</English>
```

Comment:
Leaned doesn't sound right, at least to me.
```suggestion
            <English>Dead (Leaning)</English>
```

Reply:
Changed Wounded (Leaned) and Dead (Leaned) to Leaning. Thanks.